src/grants/index.js

```diff
@@ -19,7 +19,9 @@ export const grantsPlugin = {
         },
         response: {
           status: {
-            201: Joi.object({ code: schemas.grantCode }),
+            201: Joi.object({ code: schemas.grantCode }).label(
+              "GrantCodeResponse",
```

Comment:
```suggestion
              "CreateGrantResponse",
```
so we can add other things in the future